ynput/OpenPype
# Global, Nuke: Workfile first version with template processing

## Brief description
Supporting new template workfile builder with toggle for creation of first version of workfile in case there is none yet. 

## Additional info
Since the function in abstract class was using only host related functionality for workfile saving, we do not need to implement it with abstract class functionality. 

## Testing notes:
0. Download 
[template_workfile_create.zip](https://github.com/ynput/OpenPype/files/10900186/template_workfile_create.zip)

1. Create template with placeholders and add preset to Nuke `project_settings/nuke/templated_workfile_build` as it is in example
![image](https://user-images.githubusercontent.com/40640033/223163920-d5f023a4-f133-48c1-b8be-73fcf1c5e8f5.png)

2. make sure in Anim task is no workfile yet.
3. Open Nuke in Anim task and see that it is having applied tempate in the workfile.

## Dependencies
Already merged https://github.com/ynput/OpenPype/pull/4545 within this PR
